packages/teams-js/CHANGELOG.md

```diff
@@ -1,9 +1,51 @@
 # Change Log - @microsoft/teams-js
 
-This log was last generated on Tue, 19 Apr 2022 16:08:56 GMT and should not be manually modified.
+This log was last generated on Thu, 28 Apr 2022 18:25:41 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## 2.0.0-beta.6
+
+Thu, 28 Apr 2022 18:25:41 GMT
+
+### Major changes
+
+- Updated `media` namespace to work as it did in v1 along with necessary changes to unit tests and teams-test-app
+- Adding isSupported check to people capability
```

Comment:
I think we wanted to aggregate the isSupported check changes

Reply:
Done!